bin/dfx-canister-url.test

```diff
@@ -207,6 +231,28 @@ print_title() {
     } >&2
   done
 )
+(
+  print_title "The mainnet API URLs should ignore the raw flag"
```

Comment:
Wouldn't it be better for the script to fail if you specify `--raw` for API URLs, if that isn't meaningful?

Reply:
I agree.  Done.